tests/run-make/extern-flag-fun/rmake.rs

```diff
@@ -0,0 +1,36 @@
+// The --extern flag can override the default crate search of
+// the compiler and directly fetch a given path. There are a few rules
+// to follow: for example, there can't be more than one rlib, the crates must
+// be valid ("no-exist" in this test), and private crates can't be loaded
+// as non-private. This test checks that these rules are enforced.
+// See https://github.com/rust-lang/rust/pull/15319
+
+//@ ignore-cross-compile
```

Comment:
Question: AFAICT this does not need to be `ignore-cross-compile` because we don't run the binaries.